### ISSUE TITLE
Invisible combo box items.

### ISSUE THREAD

**Issue by Wisser** [JFormDesigner/FlatLaf#426]
After updating to version 1.6.3, selected items of a JComboBox are invisible.

![grafik](https://user-images.githubusercontent.com/861721/141974719-102e67df-c647-4882-976d-efa817e7fbcb.png)

The JComboBox has the following renderer:
```
new DefaultListCellRenderer() {
  @Override
  public Component getListCellRendererComponent(JList list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
    return super.getListCellRendererComponent(list, ((IsolationLevel) value).getDisplayName(), index, isSelected, cellHasFocus);
  }
}
```

Maybe the direct use of DefaultListCellRender is not allowed like this? Nevertheless, with 1.6.1 and other L&F it works.

**Comment by DevCharly** [comment]
Damn, not again. 😢 

Occurs only when using `DefaultListCellRenderer` as renderer because is overrides `isOpaque()`...
Does not occur for default renderer, which is a `BasicComboBoxRenderer`.

> Maybe the direct use of DefaultListCellRender is not allowed like this?

It is allowed 😉 

Fix is coming...

**Comment by DevCharly** [comment]
Fixed in FlatLaf 1.6.4.

`FlatComboBoxUI.paintCurrentValue()` now temporary changes the opacity of the renderer
in the same way as done in Nimbus L&F (in `SynthComboBoxUI`).

Many thanks for reporting 👍
